tool/gravity/cli/status.go

```diff
@@ -385,6 +385,12 @@ func printNodeStatus(node statusapi.ClusterServer, w io.Writer) {
 		description = fmt.Sprintf("%v / %v", description, node.Profile)
 	}
 	fmt.Fprintf(w, "        * %v / %v\n", unknownFallback(node.Hostname), description)
+	if node.InstanceID != "" {
```

Comment:
Sorry for adding new requirements, but since you're here anyway, let's also add the information about the cloud provider to gravity status as well? It would be helpful to see. I.e. if the cluster is deployed with cloud integration enabled (aws/gce), let's display this somewhere in status output too. The provider should be a part of the Site object so chances are it is already available somewhere here, or can be easily added same way.

Reply:
I will add it above the Instance ID field:
```
        * ip-172-31-74-157 / 172.31.74.157 / general
            Provider:              aws
            Instance ID:         i-0d2f0c2ab4385ced3
            Instance Type:     x.large
```

Reply:
@lenko-d It doesn't have to be per individual node, we don't support nodes on different providers. Let's just add it somewhere in the "main" status section, maybe after cluster metadata:

```
ubuntu@node-2:~/incremental/robot-shop-1.2.0$ sudo gravity status
Cluster name:		test
Cluster status:		active
Cluster image:		robot-shop, version 1.2.0
Cloud provider:         AWS/GCE
```

Reply:
Should the provider be shown in the JSON output? Maybe like this:
```
  "cluster": {
    "application": {
      "repository": "gravitational.io",
      "name": "wordpress",
      "version": "0.0.1"
    },
    "state": "active",
    "domain": "peacefullalande5660",
    "cloud_provider": "aws",
    ...

```

Reply:
Yes, it should be shown in the json/yaml output as well.